.github/workflows/ci.yml

```diff
@@ -203,7 +203,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: ['3.8', '3.12']
+        include:
+          - python: '3.9'
+            cython: '2'
+          - python: '3.9'
+            cython: '3'
+          - python: '3.11'
+            cython: '2'
+          - python: '3.13'
+            cython: '3'
```

Comment:
add comment[s] explaining the version choices

Reply:
Specifically 3.11 is the last version Cy2 supports, while 3.9/3.13 are what we support. Also, if we set `cython: "<3"` and `>3`, I think we could avoid needing the if-else conditional later. It would make the variable a bit less clear though.

Reply:
Added comment in 0b31dfc0

Reply:
Got rid of `if` statement in 507dee3f